From face404a5792067494cfbad93b53bed01d7a6e1f Mon Sep 17 00:00:00 2001
From: Spiros Evangelatos <sevangelatos@gmail.com>
Date: Sun, 19 May 2019 00:49:25 +0300
Subject: [PATCH] Patch cares.cmake to point to system cares lib

---
 cmake/cares.cmake | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/cmake/cares.cmake b/cmake/cares.cmake
index 4ea0d8725d..8e6cb15ee7 100644
--- a/cmake/cares.cmake
+++ b/cmake/cares.cmake
@@ -35,10 +35,10 @@ if("${gRPC_CARES_PROVIDER}" STREQUAL "module")
   endif()
 elseif("${gRPC_CARES_PROVIDER}" STREQUAL "package")
   # Use "CONFIG" as there is no built-in cmake module for c-ares.
-  find_package(c-ares REQUIRED CONFIG)
-  if(TARGET c-ares::cares)
-    set(_gRPC_CARES_LIBRARIES c-ares::cares)
-    set(_gRPC_CARES_INCLUDE_DIR ${c-ares_INCLUDE_DIR})
-  endif()
-  set(_gRPC_FIND_CARES "if(NOT c-ares_FOUND)\n  find_package(c-ares CONFIG)\nendif()")
+  #find_package(c-ares REQUIRED CONFIG)
+  #if(TARGET c-ares::cares)
+    set(_gRPC_CARES_LIBRARIES cares)
+    set(_gRPC_CARES_INCLUDE_DIR /usr/include)
+    #endif()
+  #set(_gRPC_FIND_CARES "if(NOT c-ares_FOUND)\n  find_package(c-ares CONFIG)\nendif()")
 endif()
-- 
2.17.1

